CMakeLists.txt

```diff
@@ -10,14 +10,14 @@ include(cmake/Sanitizers.cmake)
 enable_sanitizers(project_options)
 
 # find dependencies
-find_package(ament_cmake REQUIRED)
 find_package(fmt REQUIRED)
 find_package(range-v3 REQUIRED)
 
 add_library(${PROJECT_NAME} INTERFACE)
```

Comment:
I would recommend against using `{PROJECT_NAME}` here since it has very poor readability and `grep`ability. Just type out `fp`. This has no benefit other than making it easier to copy-paste code between projects which isn't something you should be doing anyways.